spec/forms/developer_search_form.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe DeveloperSearchForm do
+  describe '#search' do
+    before do
+      @devs = Developer.create([{ email: 'trung@gmail.com' }, { email: 'test@gmail.com' }])
+      @pro_lang = ProgrammingLanguage.create([{ name: 'Ruby' }, { name: 'PHP' }])
+      @lang = Language.create([{ code: 'en' }, { code: 'ja' }])
+
+      @dev_lang = DeveloperLanguage.create([{ language_id: @pro_lang.first.id, developer_id: @devs.first.id }])
+      @dev_pro_lang = DeveloperProgrammingLanguage.create([{ programming_language_id: @pro_lang.first.id, developer_id: @devs.first.id }])
+
+    end
+
+    context 'with no params' do
+      it 'returns all users' do
+        res = DeveloperSearchForm.new
```

Comment:
@ntrung2006 
You should change `res` to ex: `developer_search_form` or something else